app/engine/engine.test.ts

```diff
@@ -6,6 +6,7 @@ import {
   CANT_BUZZ_FLAG,
   CLUE_TIMEOUT_MS,
   gameEngine,
+  getWinningBuzzer,
```

Comment:
commit msg again: can you reword to:

```
When two players exactly tie (in milliseconds) on a buzz,
their clients can end up in a state where each client believes it has
won the buzz.

The getWinningBuzzer function iterates through the buzzes map to
find a winner. Since JavaScript Maps iterate in insertion order, each local client can choose a different winner.

Sort the buzzes by userId prior to searching to make the winner deterministic across clients.
```